Generals/Code/GameEngine/Include/Common/MessageStream.h

```diff
@@ -270,6 +270,7 @@ class GameMessage : public MemoryPoolObject
 		MSG_META_TOGGLE_FAST_FORWARD_REPLAY,				///< Toggle the fast forward feature
 		MSG_META_TOGGLE_PAUSE,											///< TheSuperHackers @feature Toggle game pause (in replay playbacks)
 		MSG_META_STEP_FRAME,												///< TheSuperHackers @feature Step one frame (in replay playbacks)
+		MSG_META_SELECT_IDLE_WORKER,                        ///< TheSuperHackers @feature L3-M 03/08/2025 Select next idle worker
```

Comment:
Better name this `MSG_META_SELECT_NEXT_IDLE_WORKER` in case there ever will be a `MSG_META_SELECT_PREV_IDLE_WORKER`